docs/storybook/src/deprecated/Icon.stories.tsx

```diff
@@ -50,7 +49,7 @@ export const ReactNode: Story = {
 
 export const SVGPath: Story = {
   args: {
-    iconSpec: placeholderSvg,
+    iconSpec: <SvgPlaceholder />,
```

Comment:
This specifically tests the SVG loaded via href. Let's leave it as is for now. If we need to remove the `@bentley/icons-generic` dependency, we can simply add an `svg` file to the storybook.

Reply:
Done as suggested.